aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -86,6 +234,16 @@ class AztecToolbar : FrameLayout, OnMenuItemClickListener {
                 highlightAppliedStyles(selStart, selEnd)
             }
         })
+        this.editor!!.setOnKeyListener {
```

Comment:
Hey @theck13 , thanks for the change from `MainActivity` to Aztec/Formatbar.

My only concern atm is that this listener set is kinda "magic", in the sense that the toolbar view is essentially "stealing" the key click from another view (the Aztec EditText's in this case), see https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/view/View.java#9952. I think this is less readable and easier to break.

May I suggest a slight variation: 
* Introduce a `setToolbar(toolbar: AztecToolbar)` method in AztecText. That will be used by MainActivity to offer AztecText a reference to the toolbar
* Override `AztecText.onKeyUp()` and call `toolbar.onKeyUp` directly if a toolbar is set. Return true if the toolbar handled the event, otherwise just call the super.

The idea with this variation is to be able to see the key override while reading the AztecText code or when trying to override the keyUp event. Let me know what you think, thanks!